Extensions/xinbox.js

```diff
@@ -92,6 +92,22 @@ XKit.extensions.xinbox = new Object({
 			default: true,
 			value: true
 		},
+		"color_code_asks": {
+			text: "color code asks and submissions based on what blog they're for",
```

Comment:
We should probably capitalize the descriptions to match the style of the rest of the options.